app/views/course/assessment/answer/programming/_programming.json.jbuilder

```diff
@@ -77,7 +77,8 @@ json.testCases do
     end
   end
 
-  if can_read_tests && auto_grading && auto_grading.exit_code && auto_grading.exit_code != 0
```

Comment:
Suggest that we move this into a helper function?

Reply:
Done. I'm not really sure it's helpful in this case though, so I left it as a separate commit in case we choose to undo it.

It's just hiding the same logic behind a shorter name, except it's now necessary to dig up another file to find out what it's doing.

Reply:
After discussion, I should have said to abstract this to someplace else - agree that leaving it in the file will be easier to find, so this fix looks good! 👍